service/src/main/java/org/apache/griffin/core/job/repo/JobInstanceRepo.java

```diff
@@ -53,4 +53,6 @@ Licensed to the Apache Software Foundation (ASF) under one
 
     @Query("select DISTINCT s from JobInstanceBean s where s.state in ?1")
     List<JobInstanceBean> findByActiveState(State[] states);
+
+    List<JobInstanceBean> findByTriggerKey(String triggerKey);
```

Comment:
let make sure we have all indexes in place

Reply:
@chemikadze I've added `@Index` annotation, however it will only create index when the table is created - not when it is just modified by adding column. This should be ok, though.